navigation/MyStack.tsx

```diff
@@ -1,30 +1,34 @@
-import React from 'react';
-import { createStackNavigator } from '@react-navigation/stack';
-import { Home } from '../screens/Home/Home';
-import { SignUp } from '../screens/SignUp/SignUp';
-import Header from '../shared/Header';
-import SignUpHeader from '../shared/SignUpHeader';
+import React from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { HomeCard } from "../screens/Home/HomeCard";
```

Comment:
if it's a screen, then rather than call it `HomeCard`, we should call it `HomeScreen.`

Reply:
What does MyStack mean? use a name that is more meaningful